.github/workflows/ci.yml

```diff
@@ -49,14 +27,14 @@ jobs:
         uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
 
       - name: Set up environment
-        uses: mamba-org/setup-micromamba@f8b8a1e23a26f60a44c853292711bacfd3eac822 # v1.9.0
+        uses: mamba-org/setup-micromamba@617811f69075e3fd3ae68ca64220ad065877f246 # v2.0.0
         with:
           environment-file: environment.yml
           create-args: python=${{ matrix.python-version }}
           cache-environment: true
 
       - name: Install pip requirements
-        run: pip install ".[dev]"
+        run: pip install uv && uv pip install --system ".[dev]"
```

Comment:
@elephaint you legend, this is the first time I have ever seen anybody use `uv` in the wild. Have you had much experience using it and would you recommend it? I want to give it a try over `poetry` and `pyenv`, but I want to avoid any potential headaches on work projects!

Reply:
@jmoralez came with this; it's a rust-based package manager that seems mainly very much faster than pip / conda.